distributedlog-core/src/main/java/org/apache/distributedlog/tools/DistributedLogTool.java

```diff
@@ -2854,7 +2856,7 @@ public DistributedLogTool() {
         addCommand(new ListCommand());
         addCommand(new ReadLastConfirmedCommand());
         addCommand(new ReadEntriesCommand());
-        addCommand(new RecoverCommand());
+        // addCommand(new RecoverCommand());
```

Comment:
Would you mind explaining why we need to remove this? 
Also i think it would be preferable to link to a fix ticket here.